docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '3.8'
+
+services:
+  app:
+    build:
+      dockerfile: docker/Dockerfile
```

Comment:
There needs to be a`context` key here:
```
  build:
    context: .
    dockerfile: docker/Dockerfile
```

We probably could build the images with a context of `./docker`, avoiding the need for the `.dockerignore`, the paths in the `dockerfile` keys and the `ADD` line in the `Dockerfile-postges`, but it doesn't make any practical difference :shrug:

Reply:
Not sure why I took that out, guess I assumed the working directory would be the default context